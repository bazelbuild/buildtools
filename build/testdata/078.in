manifest = [
   # buildifier: table sort 2
   ("", "", ""), ("ab", "bc", "cd"), ("a", "b", "c")
]
