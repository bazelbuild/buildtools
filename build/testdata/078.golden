manifest = [
    # buildifier: table sort 2
    ("",      "",      ""),
    ("a",     "b",     "c"),
    ("ab",    "bc",    "cd"),
]
